steps/scan-with-appsweep/1.0.0/step.yml

```diff
@@ -0,0 +1,60 @@
+title: AppSweep Mobile Security Testing
+summary: Scan your Android app using Guardsquare's mobile application security testing
+  tool AppSweep
+description: "The build of your app is automatically uploaded to [AppSweep](https://appsweep.guardsquare.com)
+  for scanning. \n\nAppSweep is Guardsquare's free tool enabling developers to put
+  mobile security at the forefront of app development. AppSweep helps you identify
+  and fix security issues in your code and dependencies with actionable recommendations
+  and insights that will help you build more secure mobile apps.\n"
+website: https://www.guardsquare.com/appsweep-mobile-application-security-testing
+source_code_url: https://github.com/Guardsquare/bitrise-step-upload-to-appsweep
+support_url: https://github.com/Guardsquare/bitrise-step-upload-to-appsweep/issues
+published_at: 2022-01-18T12:13:50.297679995+01:00
+source:
+  git: https://github.com/Guardsquare/bitrise-step-scan-with-appsweep.git
+  commit: 85d7c53e690e51a6ed2071852f57382619057d50
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: true
+inputs:
+- appsweep_api_key: $APPSWEEP_API_KEY
+  opts:
+    category: Config
+    description: |
+      An `APPSWEEP_API_KEY` must be set to allow scanning of the app inside an AppSweep project. You can generate it in the API Keys section of your project settings.
+    is_required: true
+    is_sensitive: true
+    title: AppSweep API key
+- debug: false
+  opts:
+    category: Config
+    description: |
+      Set to true to upload the debug version of your app, or to false (default) to upload the release version.
+    is_required: true
+    title: Should debug version be uploaded
+- opts:
+    category: Config
+    description: |
+      Set this to the location of your project inside your repository.
```

Comment:
It's not clear from the title and description what this path should point to. Is it the path to the `build.gradle` file, or just the folder where the `build.gradle` file is?